spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -957,7 +966,9 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
 
   test("first/last") {
     withSQLConf(
+      SQLConf.COALESCE_PARTITIONS_ENABLED.key -> "true",
       CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
+      CometConf.COMET_SHUFFLE_ENFORCE_MODE_ENABLED.key -> "true",
```

Comment:
These tests are sensitive to partitioning, once coalesce partition is disabled, they won't produce same results as Spark. So I added the `COMET_SHUFFLE_ENFORCE_MODE_ENABLED` config to enforce Comet shuffle even `COALESCE_PARTITIONS_ENABLED` is enabled.